src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -97,8 +104,13 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                            + "comment text: " + feedbackResponseComment.commentText.getValue();
         }
 
-        data.comment = feedbackResponseComment;
-
+        data.comment = updatedComment;
```

Comment:
We use `feedbackResponseComment` `createdAt` time to set the value of `lastEditedAt` time of actual comment, so if we used `feedbackResponseComment` `createdAt` and `lastEditedAt` will be equal.